docs/Interesting-Taps-and-Forks.md

```diff
@@ -3,15 +3,9 @@
 A _tap_ is homebrew-speak for a Git repository containing extra formulae.
 Homebrew has the capability to add (and remove) multiple taps to your local installation with the `brew tap` and `brew untap` commands. Type `man brew` in your Terminal. The main repository [https://github.com/Homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core), often called `homebrew/core`, is always built-in.
 
-## Main taps
-
-*   [homebrew/php](https://github.com/Homebrew/homebrew-php): Repository for PHP-related formulae.
-
-`brew search` looks in these taps as well as in [homebrew/core](https://github.com/Homebrew/homebrew-core) so don't worry about missing stuff.
-
-You can be added as a maintainer for one of the Homebrew organization taps and aid the project! If you are interested please feel free to ask in an issue or pull request after submitting multiple high-quality pull requests. We want your help!
+Your taps are Git repositories located at `$(brew --repository)/Library/Taps`.
```

Comment:
Taps aren't necessarily Git repositories.  But they usually are, so this is probably OK.